content/root/quick-start-guide.textile

```diff
@@ -173,8 +173,7 @@ ably.connection.on(ConnectionState.connected, new ConnectionStateListener()
   }
 });
 
-bc[ruby].
-EventMachine.run do
+bc[ruby]. EventMachine.run do
```

Comment:
please check this. i might be wrong but i think the `EventMachine.run do` should be on a new line

Reply:
I changed it to match how the neighbouring codeblocks were laid out so it should be OK. Reviewing locally for that branch looks good:

![image](https://user-images.githubusercontent.com/25511700/103528193-6fe2a380-4e7b-11eb-9776-77554425f5c6.png)